include/klee/Config/config.h.cmin

```diff
@@ -0,0 +1,118 @@
+#ifndef KLEE_CONFIG_CONFIG_H
+#define KLEE_CONFIG_CONFIG_H
+
+/* Enable KLEE DEBUG checks */
+#cmakedefine ENABLE_KLEE_DEBUG @ENABLE_KLEE_DEBUG@
```

Comment:
You need to detect if LLVM was built with asserts and then set this macro appropriately. See `autoconf/configure.ac`.